src/momento/internal/_utilities/_grpc_channel_options.py

```diff
@@ -8,11 +8,14 @@
 DEFAULT_MAX_MESSAGE_SIZE = 5_243_000  # bytes
 
 
-def grpc_channel_options_from_grpc_config(grpc_config: GrpcConfiguration) -> grpc.aio.ChannelArgumentType:
+def grpc_channel_options_from_grpc_config(
+    grpc_config: GrpcConfiguration, is_control_client: bool = False
+) -> grpc.aio.ChannelArgumentType:
```

Comment:
Is it not an issue that this is scoped under the `aio` package but we're using it for both the async and sync clients?

Reply:
I'm not sure, it seems like it's just a type alias but and `grpc.secure_channel` in the sync code takes `Any | None` for options, but yeah idk how it resolves. Will do some local testing

> `(type alias) ChannelArgumentType: type[Sequence[Tuple[str, Any]]]`